src/sentry/runner/initializer.py

```diff
@@ -230,6 +242,19 @@ def apply_legacy_settings(settings):
                       "Use SENTRY_OPTIONS instead, key 'system.rate-limit'", DeprecationWarning)
         settings.SENTRY_OPTIONS['system.rate-limit'] = settings.SENTRY_SYSTEM_MAX_EVENTS_PER_MINUTE
 
+    if 'redis.clusters' in settings.SENTRY_OPTIONS:
+        if hasattr(settings, 'SENTRY_REDIS_OPTIONS'):
+            raise Exception("Cannot specify both SENTRY_OPTIONS['redis.clusters'] option and SENTRY_REDIS_OPTIONS setting.")
+        # TODO: This probably needs to validate that this configuration is valid somewhere?
```

Comment:
> You referring to actually pinging out to the cluster and making sure they're live? I think that's ok how it's done now.

Nah — just that a `Cluster` can actually be constructed from the given configuration options. Since it happens lazily in `ClusterManager.get`, an issue with constructing `Cluster` happens far away from the configuration and seems like it could be frustrating tracking it back to the configuration itself, since the fact that the application started at all seems like  an implicit confirmation that the configuration was valid.